jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/SeleniumTestHelper.java

```diff
@@ -57,6 +57,9 @@ public static WebDriver createWebDriver(final String webDriverClass)   {
             dc.setCapability(CapabilityType.SUPPORTS_JAVASCRIPT, true);
             dc.setCapability(InternetExplorerDriver.INTRODUCE_FLAKINESS_BY_IGNORING_SECURITY_DOMAINS, true);
         }
+        else if(webDriverClass.equalsIgnoreCase(ORG_OPENQA_SELENIUM_IE_INTERNET_EXPLORER_DRIVER)) {
```

Comment:
Please put the else if statement after '}' so it is uniform with the rest of the code.